nasl/nasl_init.c

```diff
@@ -79,6 +79,7 @@ static init_func libfuncs[] = {
   {"script_xref", script_xref},
   {"script_tag", script_tag},
   {"vendor_version", nasl_vendor_version},
+  {"collect_host_stats", nasl_collect_host_stats},
```

Comment:
Question: I'm not sure if collect is the right term as it just retrieves already collected information?